hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/HFileOutputFormat2.java

```diff
@@ -165,7 +165,7 @@ protected static byte[] combineTableNameSuffix(byte[] tableName, byte[] suffix)
    * package-private for internal usage for jobs like WALPlayer which need to use features of
    * ExtendedCell.
    */
-  static final String EXTENDED_CELL_SERIALIZATION_ENABLED_KEY =
+  public static final String EXTENDED_CELL_SERIALIZATION_ENABLED_KEY =
```

Comment:
This field is becoming part of the public API. It's worth updating the comment about its usage.

Reply:
@ndimiduk I decided to annotate that as IA.Private. I know we generally prefer not to do that for fields/methods, but for better or worse there's already a strong convention of doing it in this class (11 methods annotated IA.Private). It's probably worth a larger refactor/cleanup of HFileOutputFormat2

Let me know if you strongly disagree with this approach